metadata.json

```diff
@@ -32,51 +32,31 @@
   "operatingsystem_support": [
     {
       "operatingsystem": "RedHat",
-      "operatingsystemrelease": [
```

Comment:
Can we keep the original multi-line format ? Main concern is that this is touching unrelated lines because of the format change.

Reply:
Yes - my auto-formatter does this. I added a "plugin" that keeps arrays multiline now.

Reply:
thanks. looks better now !